debian/patches/high-dpi-position-mappings.patch

```diff
@@ -0,0 +1,121 @@
+--- a/src/gui/kernel/qhighdpiscaling_p.h
```

Comment:
Can you please add tags describing where this patch came from and what it's for at the top of this file? Please see https://dep-team.pages.debian.net/deps/dep3/ for some documentation and examples on the formatting.

Thanks.

Reply:
sorry for the noise, I should have written that it was only done for prototyping (draft PR build because I cannot create branches for this repo)
Thanks for the hint anyway!